kong/plugins/zipkin/opentracing.lua

```diff
@@ -160,8 +174,8 @@ elseif subsystem == "stream" then
       tags = {
         component = "kong",
         ["span.kind"] = "server",
-        [ip_tag(forwarded_ip)] = forwarded_ip,
-        ["peer.port"] = kong.client.get_forwarded_port(),
+        ["kong.consumer." .. ip_tag(forwarded_ip)] = forwarded_ip,
```

Comment:
I am not a lua expert. Is this going to be dynamic tag naming?

Reply:
I am going to remove this as per the conversation with @adriancole, but for the sake of completion: this would add a tag called either `"kong.consumer.ipv4"` or `"kong.consumer.ipv6"`, depending on the kind of IP.

Reply:
I see. Good to know.